java/gradle/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE=openjdk:8-jdk-alpine
+FROM ${BASE_IMAGE}
+
+RUN apk add --update --no-cache curl tar bash
```

Comment:
Do we need curl, tar, and bash when people use the build step? Otherwise you could merge this RUN into the one that downloads and installs gradle, and delete it in the same RUN.